Assets/UnityIO.cs

```diff
@@ -0,0 +1,48 @@
+using UnityEngine;
+using System;
+using System.IO;
+using System.Collections;
+
+public class UnityIO
+{
+	public class Response {
+		public string error;
```

Comment:
According to https://docs.unity3d.com/ScriptReference/WWW-error.html it could be either empty or null depending on the platform. One of your code branches also sets it to null. I recommend normalizing the "success" case to always return one value, whichever makes more sense to you. And noting that value with comments, of course.